README.md

```diff
@@ -108,3 +108,5 @@ In case the extension is not working, check the following:
 
 This project is licensed according to the terms of the Mozilla Public
 License, v. 2.0. A copy of the license is provided in `LICENSES/MPL-2.0.txt`.
+We further use icons from the Material Design Icons, which are licensed under the Apache 2.0 License.
+A copy of the license is provided in `LICENSES/Apache-2.0.txt`.
```

Comment:
Adding yet another license just for some simple icon - is that really worth it? Have you studied what the result of the combination could be license-wise?

Reply:
These licenses can be combined without any problems. Anyways, I also thought about if that is really necessary. Let's see if we can find a MIT licensed icon library and just use that instead.

Reply:
I replaced the icon with one from the yandex-ui-icons collection, which is MPL-2.0 licensed.